user-guide/Standard_Apps/PA/PA concepts/Process_Definition.md

```diff
@@ -0,0 +1,21 @@
+---
+uid: Process_Definition
+---
+
+# Process definition
+
+All [activities](xref:PA_Activities) that a process comprises of are defined in a process definition. The process definition also indicates how they are connected with each other.
```

Comment:
"comprises of" needs to be rephrased (maybe simply "all activities in a process"?)